### ISSUE TITLE
@storybook/addon-info skips PropTypes.object

### ISSUE THREAD

**Issue by lmenus** [storybooks/storybook#3130]
### Issue details

When displaying Component information with `@storybook/addon-info`, we can see its accepted and default props. To define these, we would use [Prop-Types](https://reactjs.org/docs/typechecking-with-proptypes.html). One of the available types is `PropTypes.object`. When used in a Component, `@storybook/addon-info` will not display it in the PropTypes table, we get empty DOM node (table cell) instead.

Other types render fine, just this one seems to get skipped.

### Steps to reproduce

Create a component like in the example below and render it with the addon.

### Please specify which version of Storybook and optionally any affected addons that you're running

- `@storybook/addon-info: 3.3.14`
- `@storybook/addons: 3.3.14`
- `@storybook/react: 3.3.14`

### Screenshots / Screencast / Code Snippets (Optional)

```js
import React, { Component } from 'react';
import PropTypes from 'prop-types';

class ExampleComponent extends Component {
  render() {
    return <div>{this.props.children}</div>
  }
}

ExampleComponent.propTypes = {
  thisWillBeBlank: PropTypes.object,
};

export default ExampleComponent;
```

**Comment by Hypnosphi** [comment]
I can confirm this can be reproduced in official example:
http://storybooks-official.netlify.com/?selectedKind=Addons%7CInfo.React%20Docgen&selectedStory=Comments%20from%20PropType%20declarations&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel

BTW, is there any particular reason not to use `propTypes.shape` in your case?

**Comment by lmenus** [comment]
Not really @Hypnosphi. And thank you!

**Comment by lmenus** [comment]
@CLL80 Good find! I hope @Hypnosphi says yes!

**Comment by Hypnosphi** [comment]
> Please let me know if a PR to fix this issue would be accepted.

Sure, you're welcome

**Comment by Hypnosphi** [comment]
Released as [`3.4.0-rc.4`](https://github.com/storybooks/storybook/releases/tag/v3.4.0-rc.4)
